pycbc/waveform/ringdown.py

```diff
@@ -501,20 +738,36 @@ def multimode_base(input_params, domain, freq_tau_approximant=False):
         for mode in taus:
             if 'delta_tau{}'.format(mode) in input_params:
                 taus[mode] += input_params['delta_tau{}'.format(mode)] * taus[mode]
-
     if domain == 'td':
         outplus, outcross = td_output_vector(freqs, taus,
                             input_params['taper'], input_params['delta_t'],
                             input_params['t_final'])
         for lmn in freqs:
-            hplus, hcross = td_damped_sinusoid(freqs[lmn], taus[lmn],
-                            amps[lmn], phis[lmn], outplus.delta_t,
-                            outplus.sample_times[-1], int(lmn[0]), int(lmn[1]),
-                            input_params['inclination'])
+            #hplus, hcross = td_damped_sinusoid(freqs[lmn], taus[lmn],
+            #                amps[lmn], phis[lmn], outplus.delta_t,
+            #                outplus.sample_times[-1], int(lmn[0]), int(lmn[1]),
+            #                input_params['inclination'], pols[lmn])
+            if harmonics == 'spheroidal':
```

Comment:
Wouldn't the spherical harmonics also want a final_spin parameter?

Reply:
No, don't need final spin for the spherical harmonics.